crates/store/src/db/models.rs

```diff
@@ -0,0 +1,44 @@
+//! Defines models for usage with the diesel API
+
+use diesel::prelude::*;
+
+use crate::{db::schema::accounts, errors::DatabaseError};
+
+#[derive(Debug, Clone, Queryable, QueryableByName, Selectable)]
+#[diesel(table_name = accounts)]
+#[diesel(check_for_backend(diesel::sqlite::Sqlite))]
+pub struct AccountInfoRawRow {
+    pub account_id: Vec<u8>,
+    #[allow(dead_code)]
+    pub network_account_id_prefix: Option<i64>,
+    pub account_commitment: Vec<u8>,
+    pub block_num: i64,
+    pub details: Option<Vec<u8>>,
```

Comment:
Would we ever have these be strongly typed? e.g. `AccountId`, `BlockNumber`, or is that always an additional conversion step?